release/cloudbuild.yaml

```diff
@@ -53,13 +53,13 @@ steps:
       - storage
       - cp
       - output/trusted_applet.elf
-      - gs://${_TRUSTED_APPLET_BUCKET}/${TAG_NAME}/trusted_applet.elf
+      - gs://${_FIRMWARE_BUCKET}/${_FIRMWARE_COMPONENT}/${TAG_NAME}/firmware.elf
   - name: gcr.io/cloud-builders/gcloud
     args:
       - storage
       - cp
       - output/trusted_applet.sig
-      - gs://${_TRUSTED_APPLET_BUCKET}/${TAG_NAME}/trusted_applet.sig
+      - gs://${_FIRMWARE_BUCKET}/${_FIRMWARE_COMPONENT}/${TAG_NAME}/firmware.sig
```

Comment:
Is it worth calling this `trusted_applet_transparency_dev.sig` for consistency with the `-os` repo? I know we only have the one signature here, but the name you chose for this file in `-os` is nice and clear.

Reply:
Sounds good.